cmd/podman/exec.go

```diff
@@ -19,6 +19,10 @@ var (
 			Name:  "privileged",
 			Usage: "Give the process extended Linux capabilities inside the container.  The default is false",
 		},
+		cli.BoolFlag{
+			Name:  "interactive, i",
+			Usage: "Not supported.  All exec commands are interactive by default.",
```

Comment:
nit, I know I love periods and would prefer to have them in usage statements, but our other usage statements don't end with them.